docker/pypi/wmagent/install.sh

```diff
@@ -105,9 +105,9 @@ echo "-----------------------------------------------------------------------"
 stepMsg="Generating and preserving current build id"
 echo "-----------------------------------------------------------------------"
 echo "Start $stepMsg"
-echo $RANDOM |sha256sum |awk '{print $1}' > $WMA_ROOT_DIR/.dockerBuildId
-echo "WMA_BUILD_ID:`cat $WMA_ROOT_DIR/.dockerBuildId`"
-echo "WMA_BUILD_ID preserved at: $WMA_ROOT_DIR/.dockerBuildId "
+echo $RANDOM |sha256sum |awk '{print $1}' > $WMA_ROOT_DIR/.wmaBuildId
```

Comment:
Won't this reinitialize the whole deployment process whenever a new installation is performed?
This might be the case for Tier0, which is using venv for deploying the service, but not for production, as provided in your fix here: https://github.com/dmwm/CMSKubernetes/pull/1485

Reply:
No ... actually T0 should follow the same re-initialization granularity as us.  I have already rebased this PR on top of our latest  changes  in this regard